src/main/java/teammates/common/datatransfer/FeedbackSessionResponseStatus.java

```diff
@@ -88,10 +90,19 @@ public FeedbackSessionResponseStatus() {
         return noResponse;
     }
 
+    public List<String> getStudentsWhoRespondToAnyQuestion() {
+        Collections.sort(studentsResponded, compareByTeamNameStudentName);
+        return studentsResponded;
+    }
+
     public List<String> getNoResponse() {
         return noResponse;
     }
 
+    public List<String> getStudentsResponded() {
+        return studentsResponded;
+    }
```

Comment:
We have two methods with very similar names. Add header comments to explain what each one does. Also, try to rename so that the difference is clear from the method name itself.